v3.0 breaks iphone-simulator build

Hi,

I've recently upgraded the UA Xamarin iOS component to v3.0 for iOS 10 support. 
After the upgrade, I can no longer build my iOS App for iPhone simulator. Builds for real devices work though.

Here's the build error. It looks like your binding is missing some symbols for x86.
> MTOUCH: error MT5214: Native linking failed, undefined symbol: _UALocationEventSessionIDKey. This symbol was referenced by the managed member UrbanAirship.Constants.UALocationEventSessionIDKey. Please verify that all the necessary frameworks have been referenced and native libraries linked.
> MTOUCH: error MT5202: Native linking failed. Please review the build log.

The build log shows this error:
> 		Undefined symbols for architecture i386:
> 		  "_UALocationEventSessionIDKey", referenced from:
> 		     -u command line option
> 		ld: symbol(s) not found for architecture i386

Steps to reproduce:
 - create a new iOS project
 - add UA component (v3)
 - call any UA Api function, like UAirship.TakeOff()
 - try to build for iPhone simulator

I'm not 100% sure this if this is an issue with your binding. 
It might also be a Xamarin bug or an issue with my setup, but v2.2.2 worked fine for me.
I'm on Xamarin iOS 10.0.1.10

--- a/src/AirshipBindings.iOS/ApiDefinition.cs
+++ b/src/AirshipBindings.iOS/ApiDefinition.cs
@@ -150,9 +150,6 @@ namespace UrbanAirship {
 		[Field("UALocationEventVerticalAccuracyKey", "__Internal")]
 		NSString UALocationEventVerticalAccuracyKey { get; }
 
-		// extern UALocationEventAnalyticsKey *const _Nonnull UALocationEventSessionIDKey;
-		[Field("UALocationEventSessionIDKey", "__Internal")]
-		NSString UALocationEventSessionIDKey { get; }
 
 		// extern UALocationEventUpdateType *const _Nonnull UALocationEventAnalyticsType;
 		[Field("UALocationEventAnalyticsType", "__Internal")]

